Dockerfile

```diff
@@ -1,19 +1,49 @@
 FROM python:3.8-slim
 
+WORKDIR /tmp
+
+RUN set -ex \
+    && BUILD_DEPS=" \
+    curl \
+    build-essential \
+    zlib1g-dev \
+    libpng-dev \
+    libjpeg-dev \
+    pkg-config \
+    libfontconfig1-dev \
+    " \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends $BUILD_DEPS
+
+# Install poppler pdftotext, based on xpdf3 (the old version)
+RUN set -ex \
+    && curl -k https://poppler.freedesktop.org/poppler-0.57.0.tar.xz | tar xJ \
+    && chmod -R 755 ./poppler-0.57.0 \
+    && cd ./poppler-0.57.0/ \
+	&&./configure \
+	  --prefix=/tmp/poppler \
+	  --disable-shared \
+	  --enable-build-type=release \
+	  --enable-libopenjpeg=none \
+    && make install \
+    && cp /tmp/poppler/bin/pdftotext /usr/local/bin/pdftotext
```

Comment:
#54 used a multi-stage build to reduce the built image size in preparation for using it on the deployed Heroku environments. Is there a reason this approach does not use that method?